app/views/govuk_publishing_components/components/_layout_footer.html.erb

```diff
@@ -18,9 +18,15 @@
                 <% item[:items].each do |item| %>
                   <% if item[:href] && item[:text] %>
                     <li class="govuk-footer__list-item">
-                      <a class="govuk-footer__link" href="<%= item[:href] %>">
-                        <%= item[:text] %>
-                      </a>
+                      <% if item[:target].nil? %>
```

Comment:
I think these 9 lines can be simplified as:
`<%= link_to item[:text], item[:href], target: item[:target], class: "govuk-footer__link" %>`